src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.spec.ts

```diff
@@ -25,6 +25,7 @@ import { TypeSafeMatCellDef } from '../../../shared/type-safe-mat-cell/type-safe
 import { ScenarioService } from '../../../services/scenario.service';
 import { MockComponent } from 'ng-mocks';
 import { SectionLoaderComponent } from '../../../shared/section-loader/section-loader.component';
```

Comment:
I'm only asking here because I keep forgetting, but do you have any objection to us adding a few path aliases for directories like shared and services?